examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -39,11 +39,14 @@ public static void Main(string[] args)
                             builder.AddAzureAppConfiguration(options =>
                             {
                                 options.Connect(new Uri(appConfigurationEndpoint), new DefaultAzureCredential())
-                                       .Select(keyFilter: "WebDemo:*")
+
                                        .ConfigureRefresh((refreshOptions) =>
                                        {
-                                           // Indicates that all configuration should be refreshed when the given key has changed.
```

Comment:
Why delete the comment?

Reply:
Reset the change.

---
In reply to: [433595900](https://github.com/Azure/AppConfiguration/pull/329#discussion_r433595900) [](ancestors = 433595900)